source/OutfitInfoDisplay.cpp

```diff
@@ -90,6 +102,10 @@ namespace {
 		{"ion resistance", 2},
 		{"disruption resistance", 2},
 		{"slowing resistance", 2},
+		{"discharge resistance", 2},
+		{"corrosion resistance", 2},
+		{"leak resistance", 2},
+		{"burn resistance", 2},
```

Comment:
I like all the new damage types, they really add something to the game, but I'm a bit worried about all the protection/defense attributes. Can we explain a casual player the difference between `"discharge protection"`, `"discharge resistance"`, `"disruption protection"`, `"disruption resistance"`, `"piercing protection"`, `"piercing resistance"` and `"shield protection"`? (and this is just the set of attributes for shield-related defenses/protections, the player also needs to balance against all the available outfits with protections for hull, fuel, heat, energy and balance against weapons and steering/forward thrust when outfitting ships.

Reply:
> Can we explain a casual player the difference between...

Well to be honest, I expect that by the time a player even has to worry about juggling every possible resistance or protection attribute, or even just a reasonable enough subset, that they'll be knowledgeable enough in the game's mechanics to know what's best for what occasion, as these aren't exactly basic attributes and should really only be implemented on advanced tech that takes some time to get ahold of; the only such defensive stat even available to the player in game right now is ion resistance on Remnant generators (with ion protection being on the hulls of Heliarch ships, but that isn't shown to the player in game).

Reply:
I consider myself to be an experienced player and I just don't "feel" the protection and resistance attributes; It feels to me "out of lore" as if we took the various properties and just put the words "protection" and "resistance" behind it. I'm particularly unhappy with the "resistance" word, since it sounds like a "static" property to me, while it is used as an "active" property (that costs fuel, energy and generates heat).

Is it possible to get some more variation in the terminology? For example by:
- Renaming `"burn protection"` to something like `"thermal insulation"` (it is passive and reduces the effects of burning/heating without actively countering heat.)
- Renaming `"burn resistance"` to something like `"fire suppression"` or `"flame suppression"` (or some other term that we refer to when we talk about sprinklers in buildings or systems in cargo-planes to automatically put out fires).
- Keeping either `"corrosion protection"` or `"corrosion resistance"` for the passive-property (I like resistance better there) and find something more appropriate for the "active anti-corrosion" (maybe `"hull ablation"`, `"hull decontamination"` or just `"decontamination"`)?
- Rename `"leak protection"` to something like `"puncture resistance"` (as is often used for tires).
- Rename `"leak resistance"` to something like `"self sealing"`, `"leak repair"` or just `"sealing"`.
- Rename `"discharge protection"` to something like `"shield stability"`. (With as tooltip: "Higher shield stability will reduce the initial effects of discharge damage.")
- Rename `"discharge resistance"` to something like `"shield stabilization"`? (With as tooltip: "Higher shield stabilization will allow a ship to recover faster from discharge effects.")

Reply:
I’d rather not have the names for attributes be different for each damage type despite the mechanics used being the same. Right now all one needs to understand is how protection works and how resistance works, then look to the attribute name to see which damage type each applies to. If the attribute names were unique per damage type then it’d be an exercise in figuring out which unique attribute name uses which mechanic for which damage type.

Reply:
I would expect that we create a table somewhere for content-creators to show how certain damages relate to certain defensive attributes. (Similar to the table in your PR, but with the protection and resistance attributes added and things like "cooling" and "active cooling" also added in the heat row.)

For players I would actually expect them to look at Outfits (not at stats directly) and there I expect the player to be more immersed in the game if the defensive attributes are not directly named after the damage that they counter. (In the same was that `"cooling"` sounds much better to me than `"heat removal"`, even though `"heat removal"` would be a better name for the property when it comes to game mechanics).